packages/storybook-vue/package.json

```diff
@@ -17,9 +17,10 @@
     "@storybook/core-events": "^6.4.22",
     "@storybook/theming": "^6.4.22",
     "@storybook/vue": "^6.4.22",
-    "babel-loader": "^8.1.0",
+    "babel-loader": "^10.0.0",
```

Comment:
`babel-loader` 10.x is not compatible with webpack4.  Downgrade this to `8.2.3` and it will fix the build.